metron-platform/metron-parsing/metron-parsing-storm/src/test/java/org/apache/metron/parsers/bolt/WriterBoltTest.java

```diff
@@ -84,32 +86,39 @@ public SensorParserConfig getSensorParserConfig(String sensorType) {
   public void testBatchHappyPath() throws Exception {
     ParserConfigurations configurations = getConfigurations(5);
     String sensorType = "test";
+    WriterBolt bolt = spy(new WriterBolt(new WriterHandler(batchWriter), configurations, sensorType));
     List<Tuple> tuples = new ArrayList<>();
+    List<String> messageIds = new ArrayList<>();
     for(int i = 0;i < 5;++i) {
       Tuple t = mock(Tuple.class);
```

Comment:
the format string could be a static final String constant

Reply:
Done